tensorflow_transform/mappers.py

```diff
@@ -113,8 +115,11 @@ def scale_to_z_score(x, elementwise=False, name=None, output_dtype=None):
   with tf.name_scope(name, 'scale_to_z_score'):
     # x_mean will be float16, float32, or float64, depending on type of x.
     x_mean, x_var = analyzers._mean_and_var(  # pylint: disable=protected-access
-        x, reduce_instance_dims=not elementwise, output_dtype=output_dtype)
-    return (tf.cast(x, x_mean.dtype) - x_mean) / tf.sqrt(x_var)
+        x, reduce_instance_dims=not elementwise, include_nans=not impute, output_dtype=output_dtype)
+    result = (tf.cast(x, x_mean.dtype) - x_mean) / tf.sqrt(x_var)
+    if impute:
```

Comment:
do we need impute flag ? I guess it's safe to check nan (I am not sure if none/null are handled by same value: nan) and use it while mean calc...it should be default

Reply:
I agree, I would prefer it without the flag if this seems like reasonable default behavior to everyone. Happy to remove the flag